src/System.ComponentModel.TypeConverter/tests/MemberDescriptorTests.cs

```diff
@@ -10,7 +10,7 @@ namespace System.ComponentModel.Tests
     public class MemberDescriptorTests
     {
         [Fact]
-        [ActiveIssue(8606, PlatformID.AnyUnix)]
+        [ActiveIssue(8606, PlatformID.Any)]
```

Comment:
This is fine, though you can also just remove the ```, PlatformID.Any```.  By default ActiveIssue applies to everything.